lighthouse-core/formatters/table.js

```diff
@@ -78,13 +83,16 @@ class Table extends Formatter {
 
     const rows = results.map(result => {
       const cols = headingKeys.map(key => {
+        const val = result[key];
         switch (key) {
           case 'code':
-            return '`' + result[key].trim() + '`';
+            return '`' + val.trim() + '`';
           case 'lineCol':
             return `${result.line}:${result.col}`;
+          case 'isEval':
```

Comment:
Not a huge fan of accumulating audit-specific display logic here, do we anticipate `isEval` cropping up in several other places too?